src/components/UserPage/UserPage.jsx

```diff
@@ -76,7 +76,7 @@ function UserPage() {
             {allIconComponents.editUser} Edit Info
           </button>
           <div className="w-100 d-flex justify-content-center">
-          <LogOutButton className="btn btn-primary mt-2 col col-lg-6" />
+          <LogOutButton className="btn btn-primary mt-2 col col-md-9 col-lg-6" />
```

Comment:
If we are putting a logout in the navbar, we should likely just remove the logout button in the user page.  Makes it redundant when it's accessible on any page in he nav bar.

Reply:
I agree. Removed